op-guide/tune-tikv.md

```diff
@@ -26,21 +26,233 @@ Each CF also has a separate `write buffer`. You can configure the size by settin
+# RocksDB. level0-slowdown-writes-trigger and level0-stop-writes-trigger are for the flow control of RocksDB.
+# When the number of sst files of level0 reaches 4 (the default value), the sst files of level0 and the sst files
+# of level1 which overlap those of level0 will implement compression to relieve the read pressure.
+level0-slowdown-writes-trigger = 20
+
+# When the number of sst files of level0 reaches the limit of level0-stop-writes-trigger, RocksDB stalls the new
+# write operation.
+level0-stop-writes-trigger = 36
+
+# When the level1 data size reaches the limit value of max-bytes-for-level-base, the sst files of level1
+# and their overlap sst files of level2 will implement compression. The golden rule: the first reference principle
+# of setting max-bytes-for-level-base is guaranteeing that the max-bytes-for-level-base value is roughly equal to the
+# data volume of level0. Thus unnecessary compression is reduced. For example, if the compression mode is
+# "no:no:lz4:lz4:lz4:lz4:lz4", the max-bytes-for-level-base value is write-buffer-size * 4, because there is no
+# compression of level0 and level1 and the trigger condition of compression for level0 is that the number of the
+# sst files reaches 4 (the default value). When both level0 and level1 adopt compression, it is necessary to analyze
+# RocksDB logs to know the size of an sst file compressed from an mentable. For example, if this file size is 32MB,
+# the proposed value of max-bytes-for-level-base is 32MB * 4 = 128MB.
+max-bytes-for-level-base = "512MB"
+
+# The sst file size. The sst file size of level0 is influenced by the compression algorithm of write-buffer-size
+# and level0. target-file-size-base is used to control the size of a single sst file of level1-level6.
+target-file-size-base = "32MB"
+
+# When the parameter is not configured, TiKV sets the value to 40% of the system memory size. To deploy multiple
+# TiKV nodes on one physical machine, configure this parameter explicitly. Otherwise, the OOM problem might occur
+# in TiKV.
+# block-cache-size = "1GB"
+
+[rocksdb.writecf]
+# Set it the same as rocksdb.defaultcf.compression-per-level.
+compression-per-level = ["no", "no", "lz4", "lz4", "lz4", "zstd", "zstd"]
+
+# Set it the same as rocksdb.defaultcf.write-buffer-size.
+write-buffer-size = "128MB"
+max-write-buffer-number = 5
+min-write-buffer-number-to-merge = 1
+
+# Set it the same as rocksdb.defaultcf.max-bytes-for-level-base.
+max-bytes-for-level-base = "512MB"
+target-file-size-base = "32MB"
+
+# When this parameter is not configured, TiKV sets this parameter value to 15% of the system memory size. To
+# deploy multiple TiKV nodes on one physical machine, configure this parameter explicitly. The related data
+# of the version information (MVCC) and the index-related data are recorded in write CF. In business scenario,
+# if there are many single table indexes, set this parameter value higher.
```

Comment:
"In business scenario, if there are many single table indexes," -> "In scenarios that includes many single table indexes,"